src/main/java/org/recordrobotics/charger/commands/manual/ManualClaw.java

```diff
@@ -0,0 +1,46 @@
+package org.recordrobotics.charger.commands.manual;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import org.recordrobotics.charger.subsystems.Claw;
+import org.recordrobotics.charger.control.IControlInput;
+
+
+public class ManualClaw extends CommandBase {
+
+	private Claw _claw;
+	private IControlInput _controls;
+
+	private static final double TURN_SPEED = 0.3;
+
+	public ManualClaw(Claw claw, IControlInput controls) {
+		if (claw == null) {
+			throw new IllegalArgumentException("claw is null");
+		}
+		if (controls == null) {
+			throw new IllegalArgumentException("Controls is null");
+		}
+
+		_claw = claw;
+		_controls = controls;
+		addRequirements(claw);
+	}
+
+	@Override
+	public void execute() {
+		if(_controls.getClawTurn() > 0){		
+				_claw.turn(TURN_SPEED * (Claw.CLAW_CUBE - _claw.getPosition()));
+				
+		}else if(_controls.getClawTurn() < 0){
+			_claw.turn(TURN_SPEED * (Claw.CLAW_CONE - _claw.getPosition()));
```

Comment:
The claw should turn at a constant rate, I meant that we should calculate the direction of the offset, not the actual distance.

Reply:
done